Clarify references to RFC776 to address comments in https://github.com/huitema/dnsoquic/pull/100 (#112)
huitema/dnsoquic: draft-ietf-dprive-dnsoquic.md

## Connection Handling

> "DNS Transport over TCP - Implementation Requirements" {{!RFC7766}} provides
> updated guidance on DNS over TCP, some of which is applicable to DoQ. This
> section attempts to specify which and how those considerations apply to DoQ.

Reviewer: ```suggestion
"DNS Transport over TCP - Implementation Requirements" {{!RFC7766}} provides
updated guidance on the use of DNS over TCP. This section provides similar
advice on connection handling for DoQ.
```

### Connection Reuse

Historic implementations of DNS clients are known to open and close TCP

> connections for each DNS query. As outlined in section 6.2.1 of {{!RFC7766}}, to

Reviewer: ```suggestion
connections for each DNS query. As outlined in Section 6.2.1 of {{!RFC7766}}, to
```

I'd move the `{{` as well, but that can wait until Christian has updated his copy of kramdown.

Reviewer: ```suggestion
connections for each DNS query. To
```

amortise connection setup costs, both clients and servers SHOULD support
connection reuse by sending multiple queries and responses over a single
persistent QUIC connection.

> As outlined in Section 6.2.1.1 of {{!RFC7766}}, in order to achieve performance

Reviewer: ```suggestion
In order to achieve performance
```

on par with UDP, DNS clients SHOULD send their queries concurrently over the
QUIC streams on a QUIC connection. That is, when a DNS client sends multiple
queries to a server over a QUIC connection, it SHOULD NOT wait for an
outstanding reply before sending the next query.

> ### Resource Management and Idle Timeout Values

Reviewer: ```suggestion
### Resource Management
```

Proper management of established and idle connections is important to the

> healthy operation of a DNS server. An implementation of DoQ SHOULD follow best

Reviewer: ```suggestion
healthy operation of a DNS server.

An implementation of DoQ SHOULD follow best
```

practices similar to those specified for DNS over TCP {{!RFC7766}}, in
particular with regard to:

> * Concurrent Connections (Section 6.2.2), updated by Section 6.4 of {{!RFC9103}}

Reviewer: ```suggestion
* Concurrent Connections ({{Section 6.2.2 of RFC7766}}, updated by {{Section 6.4 of !RFC9103}})
```

> * Security Considerations (Section 10)

Reviewer: ```suggestion
* Security Considerations ({{Section 10 of RFC7766}})
```
